app/controllers/registrations_controller.rb

```diff
@@ -17,12 +17,31 @@ def new
     how_they_found_us_options = @role_url_slug == 'volunteer' \
       ? HowTheyFoundUsOption.where(:for_volunteer => true) \
       : HowTheyFoundUsOption.where(:for_client => true)
+    main_goals_options = @role_url_slug == 'volunteer' \
+    ? MainGoal.where(:for_volunteer => true, :displayable => true) \
+    : MainGoal.where(:for_client => true,  :displayable => true)
+    meeting_options = MeetingOption.where(:displayable => true)
+    gender_identities = GenderIdentity.where(:displayable => true)
+    ethnicity_races = EthnicityRace.where(:displayable => true)
+    age_range_options = AgeRangeOption.all
+    education_optons = EducationOption.all
+    household_income_options = HouseholdIncomeOption.all
+    occupation_type_options = OccupationTypeOption.all
```

Comment:
These are a lot of queries, I wonder if we really need a separate table for each of these options.